commands/utils/quick-pick-azure.ts

```diff
@@ -56,3 +61,122 @@ export async function quickPickACRRegistry(): Promise<Registry> {
     const registry = registries.find((currentReg): boolean => { return desired === currentReg.name });
     return registry;
 }
+
+export async function acquireResourceGroup(subscription: Subscription, resourceGroupClient: ResourceManagementClient): Promise<ResourceGroup> {
```

Comment:
>acquireResourceGroup [](start = 22, length = 20)

I think I see similar code in previous PR. Did you try to move them to the common utility class?

Reply:
Can you check with @estebanreyl to see if he also implemented some help functions to handle acquireResourceGroup, acquireSubscription, etc?